### PR TITLE
LPS-143103 Add validations in new layout modal

### PR DESCRIPTION
**Pull request by guilhermedcamacho**

### CODE REVIEW

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by guilhermedcamacho** [comment]
ci:test:relevant

**Comment by eli-rlima** [comment]
Hey @guilhermedcamacho, I removed myself from the reviewers because I'm focused on drag and drop story. I put @interaminense as reviewer because he know the context of this PR.

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectLayout.tsx [hunk 1]
```diff
@@ -14,43 +14,44 @@
...
 import ClayButton from '@clayui/button';
-import ClayForm, {ClayInput} from '@clayui/form';
+import ClayForm from '@clayui/form';
 import ClayModal, {ClayModalProvider, useModal} from '@clayui/modal';
 import React, {useEffect, useState} from 'react';
 
-import RequiredMask from './form/RequiredMask';
-
+import useForm from '../hooks/useForm';
+import Input from './form/Input';
```
Comment (interaminense) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalAddObjectLayout.tsx:22]: Add blank line after this import, please

**Review by interaminense — CHANGES_REQUESTED** [review]
LGTM, @danvb112. Just a comment and then you can move forward. Thanks!

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**